python_tests/test_python_frontend.py

```diff
@@ -1312,6 +1312,24 @@ def fusion_func(fd: FusionDefinition):
         self.assertEqual(at_out1, nvf_out[1])
         self.assertEqual(at_out2, nvf_out[2])
 
+    def test_signbit(self):
+        inputs = [
+            torch.randn(3, 4, 5, device="cuda", dtype=torch.float32),
+            torch.randn(3, 4, 5, device="cuda", dtype=torch.float32),
+        ]
+
+        def fusion_func(fd: FusionDefinition):
+            t0 = fd.from_pytorch(inputs[0])
+            t1 = fd.from_pytorch(inputs[1])
+            t2 = fd.ops.where(fd.ops.signbit(t0), -abs(t1), abs(t1))
```

Comment:
Why compare with where instead of just direct signbit? Are you just making sure it fuses correctly?

Reply:
No particular reason. This is how `copysign` is defined in thunder. 

https://github.com/Lightning-AI/lightning-thunder/blob/main/thunder/clang/__init__.py#L846